src/components/features/createOffer/OfferType.tsx

```diff
@@ -203,6 +204,10 @@ const OfferType = ({
         "/images/examples/offer_exemple_1.png"
       ],
       titleImages: ["Clickable logos Grid", "Dynamic Banner"],
+      tooltips: [
+        "This integration allows you to display a grid of clickable logos. (Example: sponsor logo grid at the bottom of the page). Each slot in the grid will display the sponsor's proposed logo redirecting to a URL. If you need to display a single static logo, you can also choose this integration.",
+        "This integration lets you display a randomly selected ad from those submitted by sponsors, with a new ad randomly selected at each request. The ad does not redirect to a URL."
```

Comment:
"The ad does not redirect to a URL." should be removed, as we have this restriction for newsletter integration only, not html